test/dspal_tester/adsp_proc/misc_tests.c

```diff
@@ -39,10 +39,10 @@ int dspal_tester_test_dspal_get_version_info()
 	char *version_string[DSPAL_MAX_LEN_VERSION_INFO_STR];
 	char *build_date_string[DSPAL_MAX_LEN_VERSION_INFO_STR];
 	char *build_time_string[DSPAL_MAX_LEN_VERSION_INFO_STR];
-
+#if defined(DSP_TYPE_ADSP)// to be enabled after SLPI support
 	dspal_get_version_info((char *)version_string, (char *)build_date_string,
 			       (char *)build_time_string);
-
+#endif
```

Comment:
The LOG_INFO should not print below if dspal_get_version_info is not called. The values will be junk. Something like

LOG_INFO("dspal_get_version_info is currently supported on ADSP only");